packages/app/src/notification/listeners/index.ts

```diff
@@ -1,6 +0,0 @@
-/**
- * Copyright (c) Mudita sp. z o.o. All rights reserved.
- * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
- */
-
-export * from "./outbox-notification.listener"
```

Comment:
I'm not sure that is a good idea to remove this logic 🤔 I see some regression, I can't see the feature from the previous sprint. The notification popup isn't displayed 🤷‍♂️ 

https://user-images.githubusercontent.com/17147149/169995301-b7af5c9a-8f62-4a96-8a39-27bdc6fa2550.mov

Reply:
@dkarski the functionality has been reverted https://github.com/mudita/mudita-center/pull/730/commits/2cd45a0540c9ddef60ffa02ac16b4f7fa16a8a94
but it doesn't resolve the issue, it is still happening. 
I guess the reason why we see it, it's an empty `entries` list which returns the outbox endpoint 
<img width="1164" alt="Screen Shot 2022-05-24 at 12 02 14 PM" src="https://user-images.githubusercontent.com/21984930/170006200-69f3d2f5-d8fe-4294-a936-09534bd5ff1d.png">